File: 백준/Bronze/2753. 윤년/윤년.cc
#include <iostream>
using namespace std;

int main(){
    int year;
    cin >> year;
    
    // 400의 배수거나, 4의 배수이면서 100의 배수가 아닌 경우 1 아니면 0 출력.
    cout << (((year % 400) == 0) || (((year % 4) == 0) && ((year % 100) != 0)));

    return 0;
}

File: 백준/Bronze/2753. 윤년/윤년.py
import sys
input = sys.stdin.readline

# 연도 입력 받기
year = int(input().strip())

if year % 4 == 0:
    if (year % 100 != 0) or (year % 400 == 0):
        print(1)
    else :
        print(0)
else:
    print(0)
